recipe/meta.yaml

```diff
@@ -18,7 +18,6 @@ requirements:
     - numpy 1.11.*
     - openblas 0.2.19|0.2.19.*
     - python
-    - r-base
```

Comment:
See https://github.com/conda-forge/conda-smithy/pull/479 for more info. The situation is far from ideal IMO but I do not have time to debug this and add a proper fix. The skips will have to do for now.